Jenkinsfile

```diff
@@ -40,39 +46,49 @@ timestamps {
       '''.stripMargin()
     }
 
-    params = [
-      string(name: 'client_git_branch', value: scmInfo.GIT_BRANCH),
-      string(name: 'client_git_commit', value: scmInfo.GIT_COMMIT),
-      string(name: 'branch', value: 'main'),
-      string(name: 'TARGET_DOCKER_TEST_IMAGE', value: 'jdbc-centos6-default'),
-      string(name: 'parent_job', value: env.JOB_NAME),
-      string(name: 'parent_build_number', value: env.BUILD_NUMBER),
-      string(name: 'timeout_value', value: '420'),
-      string(name: 'PR_Key', value: scmInfo.GIT_BRANCH.substring(3))
-    ]
-    stage('Test') {
-      try {
-        def commit_hash = "main" // default which we want to override
-        def bptp_tag = "bptp-built"
+    def commit_hash = "main" // default which we want to override
+    def bptp_tag = "bptp-built"
+    try {
         def response = authenticatedGithubCall("https://api.github.com/repos/snowflakedb/snowflake/git/ref/tags/${bptp_tag}")
         commit_hash = response.object.sha
-        // Append the bptp-built commit sha to params
-        params += [string(name: 'svn_revision', value: commit_hash)]
-      } catch(Exception e) {
-        println("Exception computing commit hash from: ${response}")
+    } catch (Exception e) {
+      e.printStackTrace()
```

Comment:
will you know why the stack trace was thrown if you don't have any comment (like above)>

Reply:
I can put some comment. But the previous version was even worse, because `response` was out of `catch` scope and it failed without any meaningful message at all ;)